src/mpid/ch4/netmod/ucx/ucx_am.c

```diff
@@ -66,3 +66,84 @@ void MPIDI_UCX_am_send_callback(void *request, ucs_status_t status)
 
     MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_UCX_AM_SEND_CALLBACK);
 }
+
+#ifdef HAS_UCP_AM_NBX
```

Comment:
nitpick: we tend to use `HAVE_` for these type of macros.

Reply:
Sure. Updated to `HAVE_UCP_AM_NBX`.